notifications_utils/recipient_validation/phone_number.py

```diff
@@ -225,6 +226,13 @@ def validate_phone_number(self, phone_number: str) -> phonenumbers.PhoneNumber:
 
         return number
 
+    @staticmethod
+    def _is_premium_number(phone_number) -> bool:
+        phonenumber_type = phonenumbers.number_type(phone_number)
+        if phonenumber_type == phonenumbers.PhoneNumberType.PREMIUM_RATE:
+            return True
+        return False
```

Comment:
Don’t think you need the `if` here:
```suggestion
        phonenumber_type = phonenumbers.number_type(phone_number)
        return phonenumber_type == phonenumbers.PhoneNumberType.PREMIUM_RATE:
```
And maybe not even the assignment:
```suggestion
        return phonenumbers.number_type(phone_number) == phonenumbers.PhoneNumberType.PREMIUM_RATE:
```

Reply:
Agreed, just pushed the refactor